src/workerd/api/modules.h

```diff
@@ -10,13 +10,51 @@
 #include <workerd/api/sockets.h>
 #include <workerd/api/unsafe.h>
 #include <workerd/api/worker-rpc.h>
+#include <workerd/io/features.h>
 #include <workerd/io/worker.h>
 #include <workerd/jsg/modules-new.h>
+#include <workerd/jsg/setup.h>
 
 #include <cloudflare/cloudflare.capnp.h>
 
 namespace workerd::api {
 
+class EnvModule final: public jsg::Object {
+ public:
+  kj::Maybe<jsg::JsObject> getCurrent(jsg::Lock& js) {
```

Comment:
Let's define these methods out-of-line, otherwise we're piling onto the compile time for workerd-api.c++.